rocAL/include/pipeline/tensor.h

```diff
@@ -303,7 +303,7 @@ class Tensor : public rocalTensor {
     void* buffer() { return _mem_handle; }
     vx_tensor handle() { return _vx_handle; }
     vx_context context() { return _context; }
-    void set_mem_handle(void* buffer) { _mem_handle = buffer; }
+    void set_mem_handle(void* buffer) override { _mem_handle = buffer; }
```

Comment:
It is better to make this function bool and return false if the caller is sending an invalid mem_handle

Reply:
I have added a check inside this function and added throw condition in case the buffer ptr passed is invalid